spec/mailers/previews/report_mailer_preview.rb

```diff
@@ -8,7 +8,7 @@ def activated
     ).activated
   end
 
-  def submitted_delivery_partner
+  def submitted_partner_organisation
```

Comment:
For my own understanding, is this rename part of:
> renaming all the methods, locale strings, partials etc. that use the role names instead
of their numeric values.

or did it sneak in this commit? No worries if it did, just want to make sure I know what's happening!

Don't worry about it i

Reply:
Yes, as far as I could identify what calls upon what, the name of this method is correlated with the partial `_submitted_delivery_partner.text.erb` also renamed in this commit to `_submitted_partner_organisation.text.erb`, which in turn is generated based on the action (`submitted`) and the role, which changed in this commit from `delivery_partner` to `partner_organisation`. 😅